pkg/cli/images.go

```diff
@@ -46,12 +47,19 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	allProjects := a.client.Options().AllProjects
 	var images []apiv1.Image
 	var image *apiv1.Image
 	tagToMatch := ""
 	repoToMatch := ""
 
+	// Image was explicitly provided
 	if len(args) == 1 {
+		c, args[0], err = parseArgGetClient(a.client, cmd, args[0])
```

Comment:
If you do end up moving the `parseProject` resource to the factory package and renaming it to `factory.ForResource`, could you use it here as well?

Reply:
Not moved :(